src/photon/photon.h

```diff
@@ -57,6 +57,8 @@ typedef struct scheduling_algorithm_state scheduling_algorithm_state;
 
 /** The state of the local scheduler. */
 typedef struct {
+  /** The script to use when starting a new worker. */
+  char *start_worker_command;
```

Comment:
do we expect the worker start command to change during the lifetime of the local scheduler? If not, I wonder if it makes sense to have it as part of the LS state.